docs/icons.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Deze file gaat over iconen _en_ illustraties. Zullen we er twee files van maken?

Daarbij moeten we iets met die illustraties die zijn ingecheckt. Ik stel voor ze door svgo te trekken, in de huidige staat kunnen we dit niet aanbieden.

Daarbij heb ik mijn sterke bedenkingen dat het aangeboden wordt als download